firestore/google/cloud/firestore_v1beta1/constants.py

```diff
@@ -15,8 +15,24 @@
 """Helpful constants to use for Google Cloud Firestore."""
 
 
-DELETE_FIELD = object()  # Sentinel object.
+class sentinel(object):
```

Comment:
@jba Can you name the class `_Sentinel` instead? (Still not sure I like this, but if it were to merge, that'd be a more preferable class name.)

@jonparrott Can you weigh in here?

Reply:
Yeah style guide is `CamelCase` for classes, with a leading underscore for private classes.